network-api/networkapi/donate/tests/customblocks/test_notice_block.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+from wagtail import rich_text
+from wagtail.blocks.struct_block import StructBlockValidationError
+from wagtail.images.tests.utils import Image, get_test_image_file
+
+from networkapi.donate.pagemodels.customblocks.notice_block import NoticeBlock
+
+
+class NoticeBlockTest(TestCase):
```

Comment:
I think these tests would benefit from a `NoticeBlockFactory`. We could also add some tests for the page with the block...

Reply:
Updated using the 2 PR's you have sent me for reference! So I have updated the `DonateHelpPage` factory and tests to account for the new `NoticeBlockFactory`. And have also added a test for the factory as well.